index.html

```diff
@@ -9,6 +9,16 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
 
     <title>Bellshade - Learn with Our Community</title>
+
+    <meta name="description" content='Learn with Our Community' />
```

Comment:
Mungkin ini digantinya gini aja

```suggestion
    <meta name="description" content="Komunitas untuk mengelola dan menulis kode untuk proyek pendidikan sumber terbuka Indonesia" />
```

Reply:
Iya gini aja, tag linenya di masukin ke tag `title` tapi untuk OG nya simpan di `og:title`